docs/software/communication/cray-mpich.md

```diff
@@ -79,6 +79,13 @@ Cray MPICH may sometimes hang on larger runs.
 
 Performance may be negatively affected by this option.
 
+#### `"cxil_map: write error"` when doing inter-node GPU-aware MPI communication
+
+The following environment variable can be set to disable gdrcopy:
```

Comment:
Maybe (gdrcopy is obviously also technically correct, but I'm thinking GPUDirect might be a better known term)?
```suggestion
The following environment variable can be set to disable GPUDirect:
```